docs/source/dev_guide/algorithms/dbscan/distributed-steps/step-6.rst

```diff
@@ -50,6 +50,9 @@ In this step, the DBSCAN algorithm has the following parameters:
        It will require up to :math:`O(|\text{sum of sizes of neighborhoods}|)` of additional memory, 
        which in worst case can be :math:`O(|\text{number of observations}|^2)`. However, in general, performance may be better.
 
+
+       .. note:: ``memorySavingMode`` could be ``false`` only on CPU.
```

Comment:
There is no implementation for distributed dbscan steps on GPU

Reply:
fixed